src/Apps/CollectorProfile/Routes/Saves2/Components/SavesHeader.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from "react"
+import { Flex, Text, Spacer, Button, Join } from "@artsy/palette"
+import { CreateNewListModalContainer } from "./CreateNewListModal"
+import { Media } from "Utils/Responsive"
+import { CreateCollectionMutationResult } from "Apps/CollectorProfile/Routes/Saves2/types"
+import { useToasts } from "@artsy/palette"
+import { useTranslation } from "react-i18next"
+
+export const SavesHeader: React.FC = () => {
+  const { t } = useTranslation()
+  const { sendToast } = useToasts()
+  const [modalIsOpened, setModalIsOpened] = useState(false)
+
+  const handleCreateNewListClick = () => {
+    setModalIsOpened(true)
+  }
+
+  const handleComplete = (_result: CreateCollectionMutationResult) => {
+    setModalIsOpened(false)
+
+    sendToast({
+      variant: "success",
+      message: t("collectorSaves.savesHeader.listCreated"),
+    })
+  }
+
+  return (
+    <>
+      <CreateNewListModalContainer
+        visible={modalIsOpened}
+        onClose={() => setModalIsOpened(false)}
+        onComplete={handleComplete}
+      />
+
+      <Join separator={<Spacer y={0.5} />}>
+        <Text variant="lg-display">
+          {t("collectorSaves.savesHeader.savedArtworks")}
+        </Text>
+
+        {/* Desktop view */}
```

Comment:
I think it's better to use [responsive props](https://palette.artsy.net/guides/responsive/) here

Reply:
done, thank you